HANDLING-REPORTS.md

```diff
@@ -0,0 +1,37 @@
+# Handling reports and escalation
+This document describes the proposed process for handling reports and escalation, along with relevant roles in the OpenJS Foundation.
+
+Reporting to the OpenJS Foundation either concerns a matter with the Foundation or is considered an escalation where insufficient resolution was provided by a project.
+
+## Channels
+* Email: report@openjsf.org
+  * This email address is for aliasing (like tsc@nodejs.org) to roles in handling reports and escalation.
+  *  We can use groups.io
+* Form: It would be easier on the reportee to have a form template to help them structure their report. The form submission could trigger an email
+* Repo: openjs/moderation (not provisioned)
+  * A form submission could open an issue in the repo, to allow folks to discuss in context
+## Confidentiality and record-keeping
+Everything reported and discussed is confidential unless explicitly stated. For any facts to be made public, the reportee’s explicit permission is needed. All reports should be recorded, together with the discussion of it. A private moderation repo would satisfy the need for recordkeeping.
+## Steps/Escalation
+1. A report is filed: This is the first step, mostly defined in Channels above.
+2. The right people are alerted: See Roles.
+3. Contact person assigned: See Roles.
+4. Report is acknowledged: The contact person responds to let the reportee know that the report is being discussed.
+5. Facts are gathered: Time is allocated to collect information in one place to make sure everyone involved has access.
+6. Facts are discussed: The facts are discussed in context. Also see What to look forward to for guidelines on appropriate response.
+7. An action to be taken is arrived at: The action to be taken is decided by consensus.
+8. Resolution offered: The resolution is offered to the reportee by the contact person. The reportee needs to accept the resolution for the matter to be resolved. If they don’t accept, go back to step 5. If there isn't a clear way to resolve the report, the report can be reviewed by the moderation team representatives from OpenJS projects to ensure that the the action and resolution are appropriate.
+The report is resolved: The report is resolved when the reportee accepts the action.
```

Comment:
Could a report also be closed due to non-communication on behalf of the original reporter? Or if the review of step 8 indicates that all that can be done has been done? Always requiring final approval from a reporter opens up the process to some level of theoretical abuse by spamming us with reports that can't be closed.

Reply:
Well spotted. Language added to address this.